samza-core/src/main/java/org/apache/samza/storage/StorageRecovery.java

```diff
@@ -63,11 +62,11 @@
 
 
 /**
- * Recovers the state storages from the changelog streams and store the storages
+ * Recovers the state storage from both the changelog streams and stores
```

Comment:
Can you please clarify this documentation change? The usage of the term "both" is unclear to me.
Is it supposed to be "Recovers the state storage from the changelog streams and stores the state in the directory provided by the users."?

Reply:
I misunderstood the previous document, updated.